data/pirate missions.txt

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2019 by Michael Zahniser
```

Comment:
You are the one who created this file, so you should attribute the copyright to yourself, not MZ. Unless you want to transfer your copyright to MZ, but that's not needed

Reply:
Done in 3a6a51e, thanks.